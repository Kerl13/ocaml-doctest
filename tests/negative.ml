(** Let us add a failing test now.

    This the the [abs] function, it always returns a non negative valueâ€¦ Right?

    {@doctest [
      # open Testlib.Negative ;;
      # abs min_int > 0 ;;
      - : bool = true
    ]}
*)
let abs x =
  if x < 0 then - x else x
