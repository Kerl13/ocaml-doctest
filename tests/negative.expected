[ERROR] File "tests/negative.ml", lines 5-9, characters 4-6:
The following test failed:
  (abs min_int) > 0
Expected:
  - : bool = true
Got:
  - : bool = false
[INFO] File "tests/negative.ml", lines 1-10, characters 0-2: 0 / 1 tests passed
Total: 0 / 1 tests passed
